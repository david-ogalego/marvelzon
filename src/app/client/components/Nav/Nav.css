.navContainer {
    position: absolute;
    top: 0;
    height: 50px;
    background-color: grey;
    width: 100%;
}

.imageLogo {
    height: 40px;
    margin: 5px 5px 5px 15px;
}