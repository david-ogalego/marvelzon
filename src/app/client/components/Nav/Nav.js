import React from 'react';
import { Link } from 'react-router-dom';
import stylesNav from './Nav.css';
import logoMarvel from '../../images/MarvelLogo.png';

const Nav = function(props) {
    return (
        <div className={stylesNav.navContainer} >
            <Link to={`/`} >
                <img className={stylesNav.imageLogo} alt="Marvelzon" src={logoMarvel} />
            </Link>
        </div>
    )
};

export default Nav;