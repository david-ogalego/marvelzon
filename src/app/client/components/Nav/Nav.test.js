import React from 'react';
import renderer from 'react-test-renderer';
import Nav from './Nav';
import { MemoryRouter } from 'react-router-dom';

test('Snapshot Nav rendered', () => {
    const treeNav = renderer.create(
            <MemoryRouter>
                <Nav />
            </MemoryRouter>
        ).toJSON();
    expect(treeNav).toMatchSnapshot();
});