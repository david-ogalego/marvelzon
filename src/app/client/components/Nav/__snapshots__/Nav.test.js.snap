// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Nav rendered 1`] = `
<div
  className="navContainer"
>
  <a
    href="/"
    onClick={[Function]}
  >
    <img
      alt="Marvelzon"
      className="imageLogo"
      src="test-file-stub"
    />
  </a>
</div>
`;
